cmd/syft/cli/attest/attest.go

```diff
@@ -214,3 +289,31 @@ func formatPredicateType(format sbom.Format) string {
 		return ""
 	}
 }
+
+type tlogUploadFn func(*client.Rekor, []byte) (*models.LogEntryAnon, error)
+
+func uploadToTlog(ctx context.Context, sv *sign.SignerVerifier, rekorURL string, upload tlogUploadFn) (*cbundle.RekorBundle, error) {
+	var rekorBytes []byte
+	// Upload the cert or the public key, depending on what we have
+	if sv.Cert != nil {
+		rekorBytes = sv.Cert
+	} else {
+		pemBytes, err := sigs.PublicKeyPem(sv, signatureoptions.WithContext(ctx))
+		if err != nil {
+			return nil, err
+		}
+		rekorBytes = pemBytes
+	}
+
+	rekorClient, err := rekor.NewClient(rekorURL)
+	if err != nil {
+		return nil, err
+	}
+	entry, err := upload(rekorClient, rekorBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Infof("tlog entry created with index: %v", *entry.LogIndex)
```

Comment:
nit: should this be debug? or if it's worthy information, should this information be elevated to the ETUI with upload progress (even if infinite spinner)?

Reply:
cc @luhring on this one. I'm on the fence about including this in the default output. I'll demote it to debug for now, but if we come up with a good reason in this thread to why it should stick around I can elevate it into the ETUI

Reply:
Does cosign show this? I don't remember for sure but I thought it **_does_** — since it's a convenient way to find the Rekor entry that was just created (i.e. after you run this, you can use a Rekor client to just to a lookup by index)